db/schema.rb

```diff
@@ -11,20 +11,20 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20160701215002) do
```

Comment:
I'd rather see a clean PR with schema.rb cleanup the mixed in with other PRs ... can you limit this to the slack changes ?